sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/CalciteQueryPlanner.java

```diff
@@ -92,8 +95,20 @@ public CalciteQueryPlanner(JdbcConnection connection, Collection<RuleSet> ruleSe
         @Override
         public QueryPlanner createPlanner(
             JdbcConnection jdbcConnection, Collection<RuleSet> ruleSets) {
+          loadBuiltinFunctions(jdbcConnection);
```

Comment:
I don't see anything like this for ZetaSQL... How do those built in functions get loaded into jdbcConnection?

Reply:
They don't. That's the point of this PR. The functions provided by `BeamBuiltinFunctionProvider` are completely unused in ZetaSQL, so I figured we don't want them in the catalog in the first place. Otherwise, when I add functions from the jdbcConnection to the ZetaSQL catalog, I will have to have some other way to filter out builtins, or else they will conflict with the ZetaSQL builtins that are actually used.